### PR TITLE
fix: no email sending for user service

### PR DESCRIPTION
**Pull request by ytvnr**
**Issue**

https://github.com/gravitee-io/issues/issues/6645

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-yrmylqaagr.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/issues-6654-no-mail-sending-service-account/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-axzcumwksy.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/issues-6654-no-mail-sending-service-account/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-hngqoycltd.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/issues-6654-no-mail-sending-service-account/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-qeqpjkfkqj.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/issues-6654-no-mail-sending-service-account/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts [hunk 1]
```diff
@@ -555,4 +555,8 @@ export class OrgSettingsUserDetailComponent implements OnInit, OnDestroy {
       )
       .subscribe(() => this.ngOnInit());
   }
+
+    isServiceUser(): boolean {
+      return !this.user.firstname;
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: a bit twisted. but if no other solution why not

**Review by ThibaudAV — APPROVED** [review]
I have tested and it seems ok. 
Except for something that is for the 2 default accounts `api1` and `admin`. it is not possible to reset their password because no firstName. 
but maybe OSF (🇫🇷 ) or they should add a firstName as it seems mandatory

File: gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts (hunk 1 above)
Comment (ytvnr) [reply gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts]: yes, @gaetanmaisse suggested me to have it in the component inside of the template to at least understand the meaning of this condition.
if this feature grow in the future, we will have to manage this kind of user with an explicit type imho

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-nnmdwksmtd.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/issues-6654-no-mail-sending-service-account/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-tmzzevduwd.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/issues-6654-no-mail-sending-service-account/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**